nat-lab/bin/symmetric-gw

```diff
@@ -4,14 +4,17 @@ set -e
 
 echo 1 > /proc/sys/net/ipv4/ip_forward
 
+public_itf=$(ip route show | grep -e '10.0.0.0/16' | egrep -o 'dev [a-z0-9]+' | cut -f 2 -d ' ')
+private_itf=$(ip route show | grep -e '192.168.' | egrep -o 'dev [a-z0-9]+' | cut -f 2 -d ' ')
```

Comment:
Same as https://github.com/NordSecurity/libtelio/pull/96/files#r1324148575

Reply:
For the `192.168.` part I will leave the old method, cause `awk` searches in all `ip a` output, and gets more complicated.

Reply:
Changed all long bash pipelines.